src/main/java/frc/robot/subsystems/shooter/Shooter.java

```diff
@@ -36,6 +39,10 @@
     private TalonFXSimCollection simCollection;
     private double currentTime = 0;
     private double lastTime = 0;
+    private static final ShuffleboardTab tab = Shuffleboard.getTab("Velocity");
```

Comment:
IMO it's better to organize Network Tables such that each subsystem has its own subtable.